### PR TITLE
Moved static files to the NGINX server

### PR DESCRIPTION
**Pull request by jope1**
This PR moves the static files for the capacity status API into the NGINX server so that they can be served from there.

As part of the proxy build process, we copy the static files from the application into the NGINX docker container. The static files are stored in /var/www/capacity-status-api on the NGINX server.  We then root NGINX to these static files when it comes across /static/ in the URL.  This solution works across all environments (local, local-containerised, AWS)

N.B. There is also a change to the location of two AWS secret stores. (These are only name changes).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,17 +31,18 @@ project-build-proxy:
 	cp -f \
 		$(PROJECT_DIR)/certificate/* \
 		$(DOCKER_DIR)/proxy/assets/certificate
+	cp -f -r \
```
Comment (stefaniuk) [inline Makefile]: Can we convert this line to use `cp -rf` so the flags are combined in one set of options?

File: build/docker/proxy/assets/etc/nginx/nginx.conf [hunk 2]
```diff
@@ -22,6 +22,12 @@ http {
         ssl_certificate      /certificate/certificate.pem;
         ssl_certificate_key  /certificate/certificate.key;
 
+        # Root to static files
+        location /static/  {
+            autoindex    on;
+            root /var/www/capacity-status-api;
```
Comment (stefaniuk) [inline build/docker/proxy/assets/etc/nginx/nginx.conf]: Can we change the directory of the static files to `/application`? This is for consistency as we do that in the other type of containers.

File: build/docker/proxy/Dockerfile [hunk 3]
```diff
@@ -3,6 +3,7 @@ FROM nginx:latest
 COPY assets/certificate/certificate.key /certificate/certificate.key
 COPY assets/certificate/certificate.pem /certificate/certificate.pem
 COPY assets/etc/nginx/nginx.conf /etc/nginx/nginx.conf
+COPY assets/var/www/capacity-status-api/static/ /var/www/capacity-status-api/static
```
Comment (stefaniuk) [inline build/docker/proxy/Dockerfile]: Can we change the directory of the static files to `assets/application`? This is for consistency as we do that in the other type of containers.

File: Makefile (hunk 1 above)
Comment (jope1) [reply Makefile]: Sure

File: build/docker/proxy/assets/etc/nginx/nginx.conf (hunk 2 above)
Comment (jope1) [reply build/docker/proxy/assets/etc/nginx/nginx.conf]: So /application/static ? or /application/capacity-status-api/static?

**Pull request merged**